[2024-03-10T14:08:50.224+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_dag.not_eligible_to_vote scheduled__2024-03-06T12:00:00+00:00 [queued]>
[2024-03-10T14:08:50.231+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_dag.not_eligible_to_vote scheduled__2024-03-06T12:00:00+00:00 [queued]>
[2024-03-10T14:08:50.231+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-10T14:08:50.260+0530] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): not_eligible_to_vote> on 2024-03-06 12:00:00+00:00
[2024-03-10T14:08:50.265+0530] {standard_task_runner.py:60} INFO - Started process 194709 to run task
[2024-03-10T14:08:50.273+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'catchup_dag', 'not_eligible_to_vote', 'scheduled__2024-03-06T12:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/execute_catchup.py', '--cfg-path', '/tmp/tmp33xatuyb']
[2024-03-10T14:08:50.276+0530] {standard_task_runner.py:88} INFO - Job 168: Subtask not_eligible_to_vote
[2024-03-10T14:08:50.332+0530] {task_command.py:423} INFO - Running <TaskInstance: catchup_dag.not_eligible_to_vote scheduled__2024-03-06T12:00:00+00:00 [running]> on host BreakThrough.
[2024-03-10T14:08:50.416+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kai' AIRFLOW_CTX_DAG_ID='catchup_dag' AIRFLOW_CTX_TASK_ID='not_eligible_to_vote' AIRFLOW_CTX_EXECUTION_DATE='2024-03-06T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-06T12:00:00+00:00'
[2024-03-10T14:08:50.417+0530] {logging_mixin.py:188} INFO - Not Eligible to Vote
[2024-03-10T14:08:50.417+0530] {python.py:202} INFO - Done. Returned value was: None
[2024-03-10T14:08:50.420+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=catchup_dag, task_id=not_eligible_to_vote, execution_date=20240306T120000, start_date=20240310T083850, end_date=20240310T083850
[2024-03-10T14:08:50.486+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-10T14:08:50.499+0530] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
